docs/usage_guide/advanced_usage/index.rst

```diff
@@ -5,18 +5,19 @@ Advanced Usage
    :titlesonly:
    :hidden:
 
+   /flow/modin/distributed/dataframe/pandas
    spreadsheets_api
    progress_bar
    modin_sql
    modin_xgboost
    modin_in_the_cloud
-   
+
 .. meta::
     :description lang=en:
         Description of Modin's advanced features.
 
-Modin aims to not only optimize pandas, but also provide a comprehensive, 
-integrated toolkit for data scientists. We are actively developing data science tools 
+Modin aims to not only optimize pandas, but also provide a comprehensive,
+integrated toolkit for data scientists. We are actively developing data science tools
 such as DataFrame - spreadsheet integration, DataFrame algebra, progress bars, SQL queries
 on DataFrames, and more. Join the `Discourse`_ for the latest updates!
```

Comment:
```suggestion
on DataFrames, and more. Join us on `Slack`_ and `Discourse`_ for the latest updates!
```
Note add link to Slack: https://modin.org/slack.html

Reply:
Added the link.